R/pkg/R/DataFrame.R

```diff
@@ -3321,3 +3328,11 @@ setMethod("randomSplit",
             }
             sapply(sdfs, dataFrame)
           })
+
+# A global singleton for an empty SparkR DataFrame.
+getEmptySparkRDataFrame <- function() {
+  if (is.null(.sparkREnv$EMPTY_DF)) {
+    .sparkREnv$EMPTY_DF <- as.DataFrame(data.frame(0))
+  }
+  return(.sparkREnv$EMPTY_DF)
```

Comment:
also, the convention seems to be `get(".sparkRsession", envir = .sparkREnv)` - with the object name in `.lowerCamalCase` - could you change to that